src/contracts/executor.ts

```diff
@@ -95,11 +95,11 @@ export class Executor extends Logger {
     }
     if (args.length && typeof args[args.length - 1] === 'object') {
       options = Object.assign(options || {}, args[args.length - 1]);
-      return contract.methods[functionName].apply(contract.methods, args.slice(0, args.length - 1)).call(options);
+      return contract.methods[functionName](...args.slice(0, args.length - 1)).call(options);
```

Comment:
I think this should also work:

```suggestion
      return contract.methods[functionName](...args.slice(0, -1)).call(options);
```